### PR TITLE
PG-208 Doc Added Contibuting guide

### PR DESCRIPTION
**Pull request by nastena1606**
new file: CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Contributing guide
+
+Welcome to `pg_stat_monitor` - the statistics collection tool for PostgreSQL!
```
Comment (LenzGr) [inline CONTRIBUTING.md:3]:
```suggestion
Welcome to `pg_stat_monitor` - the Query Performance Monitoring Tool for PostgreSQL!
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+# Contributing guide
```
Comment (LenzGr) [inline CONTRIBUTING.md:1]:
```suggestion
# Contributing Guide
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+11. Make sure your pull request contains only one commit message
+
+### Building pg_stat_monitor
+
+
+To build `pg_stat_monitor` from source code, you require the following:
+
+* git
+* make
+
```
Comment (LenzGr) [inline CONTRIBUTING.md:76]: There's probably more to it, e.g. a C compiler? @darkfronza or @ibrarahmad - can you add some more details here?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
+# Contributing guide
+
+Welcome to `pg_stat_monitor` - the statistics collection tool for PostgreSQL!
+
+We're glad that you would like to become a Percona community member and participate in keeping open source open.
+
+You can contribute in one of the following ways:
+
```
Comment (LenzGr) [inline CONTRIBUTING.md:8]: Lots of empty lines in this document, could you trim these down?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+3. [Submit a pull request (PR) with the code patch](#submit-a-pull-request)
+4. [Contribute to documentation](#contributing-to-documentation)
+
+By contributing, you agree to the [Percona Community code of conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md).
+
+
+## Submit a bug report or a feature request
+
+
+If you found a bug in `pg_stat_monitor`, please submit the report to the [Jira issue tracker](https://jira.percona.com/projects/PG/issues).
```
Comment (LenzGr) [inline CONTRIBUTING.md:22]:
```suggestion
If you would like to suggest a feature of if you found a bug in `pg_stat_monitor`, please submit the report to the [Jira issue tracker](https://jira.percona.com/projects/PG/issues).
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+3. As a general rule of thumb, try to create bug reports that are:
+
+  * Reproducible: describe the steps to reproduce the problem.
+  * Unique: check if there already exists a JIRA ticket to describe the problem.
+  * Scoped to a Single Bug: only report one bug in one JIRA ticket
+
+## Submit a pull request
+
+
+All bug reports, enhancements and feature requests are tracked in [Jira issue tracker](https://jira.percona.com/projects/PG). Though not mandatory, we encourage you to first check for a bug report among Jira issues and in the PR list: perhaps the bug has already been addressed.
```
Comment (LenzGr) [inline CONTRIBUTING.md:41]: This paragraph should be moved up, into the "Submit a bug or feature request" section.

File: CONTRIBUTING.md (hunk 1 above)
Comment (nastena1606) [reply CONTRIBUTING.md:3]: Fixed

File: CONTRIBUTING.md (hunk 4 above)
Comment (nastena1606) [reply CONTRIBUTING.md:8]: Done

File: CONTRIBUTING.md (hunk 6 above)
Comment (nastena1606) [reply CONTRIBUTING.md:41]: Amended

File: CONTRIBUTING.md (hunk 5 above)
Comment (nastena1606) [reply CONTRIBUTING.md:22]: Fixed

**Pull request merged**